packages/primitives/image/readme.md

```diff
@@ -0,0 +1,20 @@
+`Image` is an element that allowed to render an image. 
+Only the image scaling can be styled.
+
+These map to the same corresponding props in Web and in React Native.
+
+## Raw type definitions
+
+```ts
+export type TInputProps = {
+  id?: string,
+  alt?: string,
+  source: string,
+  height: number,
+  width: number,
+  objectFit?: string,
+  ImageResizeMode?: string,
```

Comment:
we should provide a single normalized API that applies both for web and native, looks like `objectFit` and `ImageResizeMode` is about the same but for different platforms? if it's so then we should pick one prop instead of two and normalize it.

Reply:
use `ResizeMode` for all! - done